packages/framework/src/Model/Product/ProductFacade.php

```diff
@@ -326,6 +341,26 @@ public function getByUuid(string $uuid): Product
         return $this->productRepository->getOneByUuid($uuid);
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductData $productData
+     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     */
+    public function editProductStockRelation(ProductData $productData, Product $product): void
```

Comment:
this method is public just because of two tests. Just saying, you can close this comment :smile:

Reply:
Although the change wouldn't be hard, I think it may be handy to be able to edit just stocks... (I'm thinking about some stock-only related import for instance).
If you feel it differently, I'm open to discussion.